solomonhawk/src/day1.rs

```diff
@@ -1,31 +1,58 @@
 #[aoc_generator(day1)]
 pub fn input_generator(input: &str) -> Vec<usize> {
-    input
-        .trim()
-        .split("\n")
-        .map(|n| n.parse().unwrap())
-        .collect()
+    input.split("\n").map(|n| n.parse().unwrap()).collect()
```

Comment:
If you want you can also always use `.expect("Some message")` in place of unwrap.  Gives the user a little more info.